src/Features/Context/TiqrContext.php

```diff
@@ -438,34 +407,34 @@ public function iScanTheTiqrAuthenticationQrcode()
     /**
      * @When /^I clear the logs$/
      */
-    public function clearTheLogs()
+    public function clearTheLogs(): void
     {
-        /** @var FileLogger $logger */
-        $logger = $this->kernel->getContainer()->get(FileLogger::class);
+        $logger = $this->fileLogger;
```

Comment:
Why use an inline var $logger?

Reply:
Historically, that was done to get code completion. Now that everything is typed. That's no longer required. Fixing this.